src/duckstation-qt/debuggerwindow.ui

```diff
@@ -366,98 +361,91 @@
   </action>
   <action name="actionStepOut">
    <property name="icon">
-    <iconset resource="resources/resources.qrc">
-     <normaloff>:/icons/debug-step-out.png</normaloff>:/icons/debug-step-out.png</iconset>
+    <iconset theme="debug-step-out-line"/>
    </property>
    <property name="text">
     <string>Step Out</string>
    </property>
    <property name="toolTip">
-    <string>Step O&amp;ut</string>
```

Comment:
These should be on the text instead of the tooltip, I'm guessing.

Reply:
That does it.

![image](https://github.com/stenzek/duckstation/assets/14798312/c6c70d0b-c3c5-4bed-b9bc-e27bd5d90e8d)